gen.go

```diff
@@ -110,6 +109,97 @@ var stringTypeSuffixes = []string{
 	"yaml",
 }
 
+var enumExamples = map[string]string{
+	"AlertSourceStatusTypeEnum":                       opslevel.AllAlertSourceStatusTypeEnum[0],
+	"ApiDocumentSourceEnum":                           opslevel.AllApiDocumentSourceEnum[0],
+	"AlertSourceTypeEnum":                             opslevel.AllAlertSourceTypeEnum[0],
```

Comment:
This feels like a chicken and egg problem since these are generated data - how does this work when enum.go doesn't exist?

Reply:
Reading further down - i think you can just replace this with data from the schema if you build it up during schema parsing first.

Reply:
I've got a simple fix for this 👍